### ISSUE TITLE
atan parentheses issue

### ISSUE THREAD

**Issue by fuyans** [erwanp/pytexit#14]
atan function generate normal `(...)` whereas other functions, e.g. tan, produces `\left(...\right)`.

To reproduce:
```
>>> from pytexit import py2tex
>>> py2tex('y=atan(x)')
$$y=\arctan(x)$$
>>> py2tex('y=tan(x)')
$$y=tan\left(x\right)$$
```

Not sure if this is intended but I personally prefer to keep the consistency, i.e. to always use \left(\right).

**Comment by phidi04** [comment]
Can be closed due to #23

**Comment by erwanp** [comment]
Done, thank you !!
